### ISSUE TITLE
X-linked Primarily Recessive inheritance

### ISSUE THREAD

**Issue by angharadroberts** [obophenotype/human-phenotype-ontology#7489]
**Preferred term label:**
X-linked Primarily Recessive inheritance

**Synonyms**

**Definition (free text, please give PubMed ID)**
X-linked inheritance in which females typically manifest a mild disease phenotype or disease expression

**Parent term (use hpo.jax.org/app)**
X-linked recessive inheritance HP:0001419

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@angharadroberts 
I have updated the term label and definition as suggested. I have made the original definition a comment and slightly modified it as follows

Comment: A mode of inheritance that is observed for recessive traits related to a gene encoded on the X chromosome. In the context of medical genetics, X-linked recessive disorders manifest in males (who have one copy of the X chromosome and are thus hemizygotes), but generally do not display clinical manifestation or display milder manifestations in female heterozygotes who have one mutant and one normal allele.
